Sources_v3/Database/DatabaseContainer.swift

```diff
@@ -82,7 +82,7 @@ class DatabaseContainer: NSPersistentContainer {
         
         persistentStoreDescriptions = [description]
                 
-        if shouldFlushOnStart {
+        if shouldFlushOnStart, kind != .inMemory {
```

Comment:
This needs to be tested. Can you take care of that @mantergo and push a commit with the missing test, please?

Reply:
Thanks for pinging, added test 🙂